packages/frontend/src/fetcher/subgraph/queries/index.ts

```diff
@@ -0,0 +1,82 @@
+export interface SubgraphQuestion {
```

Comment:
Shouldn't this type include the `responses` too? Since below we're querying the `responses` of a `question`.